Added notes about CAF applications and the addition of the MPL app. (#625)
THEOplayer/documentation: theoplayer/how-to-guides/03-cast/01-chromecast/06-enable-chromecast-on-the-sender.md

<script type="text/javascript" src="//www.gstatic.com/cv/js/sender/v1/cast_sender.js?loadCastFramework=1"></script>
```

> Once this SDK is imported, THEOplayer will automatically add the Chromecast casting capability to your player (if you've enabled the Chromecast feature for your THEOplayer SDK). THEOplayer will use its default Chromecast receiver application if you do not specify an `appID` value on the configuration.

Reviewer: ```suggestion
Once this SDK is imported, OptiView Player will automatically add the Chromecast casting capability to your player (if you've enabled the Chromecast feature for your OptiView Player SDK). OptiView Player will use its default Chromecast receiver application if you do not specify an `appID` value on the configuration.
```

## Connect with custom Chromecast Receiver application {#custom-receiver-app}

The THEOplayer SDK on your Sender application connects with THEO Chromecast Receiver application by default,

> but you can connect it to a custom Chromecast Receiver application instead. The default THEO Chromecast Receiver application has THEOplayer functionality, but uses Shaka for media playback. If you want to use Google's legacy Media Player Library (MPL) for playback, you will need to use either a custom application or connect to THEO's MPL Receiver application, using the `appID` value of `44BAE7D1`. Note that the MPL receiver does not support HLS-CMAF streams.

Reviewer: ```suggestion
but you can connect it to a custom Chromecast Receiver application instead. The default Dolby OptiView Chromecast Receiver application has OptiView Player functionality, but uses Shaka for media playback. If you want to use Google's legacy Media Player Library (MPL) for playback, you will need to use either a custom application or connect to Dolby OptiView's MPL Receiver application, using the `appID` value of `44BAE7D1`. Note that the MPL receiver does not support HLS-CMAF streams.
```

});
```

> If you do not specify an `appID` value, the player will cast to the default THEOplayer Chromecast application. This is a Receiver application that has THEOplayer functionality integrated into it, but uses the Shaka player for media playback. If you require a Receiver application that uses Google's legacy Media Player Library (MPL), there is a Receiver application with THEOplayer functionality that uses MPL. Its `appID` is `44BAE7D1`. Note that the MPL receiver does not support HLS-CMAF streams.

Reviewer: ```suggestion
If you do not specify an `appID` value, the player will cast to the default OptiView Player Chromecast application. This is a Receiver application that has OptiView Player functionality integrated into it, but uses the Shaka player for media playback. If you require a Receiver application that uses Google's legacy Media Player Library (MPL), there is a Receiver application with OptiView Player functionality that uses MPL. Its `appID` is `44BAE7D1`. Note that the MPL receiver does not support HLS-CMAF streams.
```
